Detectors/MUON/MCH/Simulation/src/Digitizer.cxx

```diff
@@ -143,6 +143,7 @@ int Digitizer::processHit(const Hit& hit, int detID, double event_time)
   bool padexists = seg.findPadPairByPosition(localX, localY, padidbendcent, padidnoncent);
   if (!padexists) {
     LOG(ERROR) << "Did not find  _any_ pad for localX,Y=" << localX << "," << localY;
```

Comment:
The two errors should be in a single line

Reply:
Ok, is changed.